scripts/manage-views.py

```diff
@@ -28,7 +28,7 @@
     parser.add_argument('-e', '--dbengloglevel', help='Database engine logging level',
                         choices=log_level_choices, default='WARNING')
     parser.add_argument('operation', help="Operation to perform",
-                        choices=['create', 'refresh'])
+                        choices=['refresh'])
```

Comment:
Probably not a big deal, but there really isn't a "choice" here anymore.  Are we expecting to have more choices in the future? If not I might suggest this `operation` argument to be changed true/false `refresh` flag.

Reply:
Good catch.